sample/src/main/kotlin/com/thefuntasty/donutsample/ui/playground/PlaygroundActivity.kt

```diff
@@ -2,6 +2,7 @@ package com.thefuntasty.donutsample.ui.playground
 
 import android.os.Bundle
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/828b548c8186e32727ef08c33dcf88e758e87083/sample/src/main/kotlin/com/thefuntasty/donutsample/ui/playground/PlaygroundActivity.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>